packages/manager/apps/telecom/src/app/telecom/pack/resiliation/pack-resiliation.controller.js

```diff
@@ -435,4 +444,68 @@ export default class PackResiliationCtrl {
       'changeOperator',
     ].includes(this.model.reason.value);
   }
+
+  /**
+   * Retrieve access name, access type and launch eligibility if not fiber access
+   */
+  checkForEligibility() {
+    this.$http
+      .get(`/pack/xdsl/${this.$stateParams.packName}/xdslAccess/services`)
+      .then(({ data }) => {
+        if (data.length > 0) {
+          const accessName = data[0];
+          this.$http.get(`/xdsl/${accessName}`).then((response) => {
+            const { accessType } = response.data;
+            if (['vdsl', 'adsl'].includes(accessType)) {
+              this.$http
+                .get(`/xdsl/${accessName}/fiberEligibilities`)
+                .then((result) => {
+                  if (result.data.length > 0) {
+                    const id = result.data[0];
+                    this.$http
+                      .get(`/xdsl/${accessName}/fiberEligibilities/${id}`)
+                      .then((res) => {
+                        this.eligibility = res.data;
+                        if (this.eligibility.status === this.ELIGIBLE) {
+                          this.isEligible = true;
+                        } else {
+                          this.isEligibleNotYet = true;
+                        }
```

Comment:
**nitpick:** Unnecessary variable. The same boolean can be used to find out if eligible or not.
```suggestion
            this.isEligible = this.eligibility.status === this.ELIGIBLE;
```

Reply:
No I need 2 different flags to display 2 different modal into HTML page. And when they are not set, no modals are displayed.

Reply:
Hello @Steffy29,
Instead of using `this.isEligibleNotYet`, why dont you use `!this. isEligible` wherever required.

Reply:
Like I explain the previous time, we have 3 status possible: eligible, not eligible and not need. So I need 2 flags, one for eligible, one for eligible and when we are in not need status, the 2 flags are to false state.